### PR TITLE
tools: add DM query status doc

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/tidb-tools/pull/153
@amyangfei @lilin90 PTAL

### CODE REVIEW

File: tools/dm-query-status.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                            "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                        ],
+                        "unresolvedGroups": [   # The sharding group that is not resolved.
+                            {
+                                "target": "`test`.`t_target`",                  # The downstream database table to be synchronized.
+                                "DDLs": [
+                                    "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                                ],
+                                "firstPos": "(bin|000001.000001, 3130)",        # The starting position of the sharding DDL statement.
+                                "synced": [                                     # The sharded table of the sharding DDL statement that `Sync` has read.
```
Comment (amyangfei) [inline tools/dm-query-status.md]: The shared table that `sync` unit has read its sharding DDL statement.
The original Chinese is not accuracy, `the shared table` does not belong to `sharding DDL`.

File: tools/dm-query-status.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                                    "`test`.`t2`"
+                                    "`test`.`t3`"
+                                    "`test`.`t1`"
+                                ],
+                                "unsynced": [                                   # The upstream table of the sharding DDL statement that has not been
```
Comment (amyangfei) [inline tools/dm-query-status.md]: The upstream table that has not executed this sharding DDL

File: tools/dm-query-status.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+                                                                                # executed. If any upstream tables have not been synchronized completely,  
+                                                                                # `blockingDDLs` is empty.
+                                ]
+                            }
+                        ],
+                        "synced": false         # Whether the progress of incremental synchronization in the downstream has been the same as that in the
```
Comment (amyangfei) [inline tools/dm-query-status.md]: In fact there is no `incremental synchronization` in upstream. what about ``Whether the incremental synchronization catches up with upstream and has the same binlog position as that in the upstream``

File: tools/dm-query-status.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+            "subTaskStatus": [
+                {
+                    "name": "test",
+                    "stage": "Running",
+                    "unit": "Load",
+                    "result": null,
+                    "unresolvedDDLLockID": "",
+                    "load": {                   # The synchronization information of the `Load` processing unit.
+                        "finishedBytes": "115", # The number of bytes that have been imported fully.
+                        "totalBytes": "452",    # The total number of bytes that need to be imported.
```
Comment (amyangfei) [inline tools/dm-query-status.md]:
```suggestion
                        "totalBytes": "452",    # The total number of bytes that need to be loaded.
```

File: tools/dm-query-status.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+                        "progress": "25.44 %"   # The progress of the full import.
```
Comment (amyangfei) [inline tools/dm-query-status.md]:
```suggestion
                        "progress": "25.44 %"   # The progress of the load process.
```

File: tools/dm-query-status.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+            "msg": "",
+            "subTaskStatus": [
+                {
+                    "name": "test",
+                    "stage": "Running",
+                    "unit": "Load",
+                    "result": null,
+                    "unresolvedDDLLockID": "",
+                    "load": {                   # The synchronization information of the `Load` processing unit.
+                        "finishedBytes": "115", # The number of bytes that have been imported fully.
```
Comment (amyangfei) [inline tools/dm-query-status.md]:
```suggestion
                        "finishedBytes": "115", # The number of bytes that have been loaded.
```

File: tools/dm-query-status.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+                        "totalBytes": "452",    # The total number of bytes that need to be imported.
```
Comment (amyangfei) [inline tools/dm-query-status.md]:
```suggestion
                        "totalBytes": "452",    # The total number of bytes that need to be loaded.
```

File: tools/dm-query-status.md [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+                        "progress": "25.44 %"   # The progress of the full import.
```
Comment (amyangfei) [inline tools/dm-query-status.md]:
```suggestion
                        "progress": "25.44 %"   # The progress of the load process.
```

File: tools/dm-query-status.md [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+                "result": null
+            }
+        }
+    ]
+}
+
+```
+
+For the status description and status switch relationship of "stage" of "subTaskStatus" of "workers", see [Subtask status](#subtask-status).
+For operation details of "unresolvedDDLLockID" of "subTaskStatus" of "workers", see [Troubleshooting Sharding DDL Locks](../tools/troubleshooting-sharding-ddl-locks/).
```
Comment (amyangfei) [inline tools/dm-query-status.md]: Where do LINE 184-185 come from, I can't find them in CN docs.
https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/task-handling/query-status.md

File: tools/dm-query-status.md (hunk 1 above)
Comment (lilin90) [reply tools/dm-query-status.md]: The suggested version is still not appropriate in grammar. Please confirm the sentence logic with @amyangfei and provide a new version. :)

File: tools/dm-query-status.md (hunk 2 above)
Comment (lilin90) [reply tools/dm-query-status.md]:
```suggestion
                                "unsynced": [                                   # The upstream table that has not executed this sharding DDL statement.
```

File: tools/dm-query-status.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+                                    "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
+                                ],
+                                "firstPos": "(bin|000001.000001, 3130)",        # The starting position of the sharding DDL statement.
+                                "synced": [                                     # The sharded table of the sharding DDL statement that `Sync` has read.
+                                    "`test`.`t2`"
+                                    "`test`.`t3`"
+                                    "`test`.`t1`"
+                                ],
+                                "unsynced": [                                   # The upstream table of the sharding DDL statement that has not been
+                                                                                # executed. If any upstream tables have not been synchronized completely,
```
Comment (lilin90) [inline tools/dm-query-status.md]:
```suggestion
                                                                                # executed. If any upstream tables have not finished synchronization,  
```

File: tools/dm-query-status.md (hunk 5 above)
Comment (CaitinChen) [reply tools/dm-query-status.md]: @amyangfei load process -> loading process

File: tools/dm-query-status.md (hunk 9 above)
Comment (CaitinChen) [reply tools/dm-query-status.md]: @amyangfei They are in the above code block. Relative URLs are not user-friendly in the code block, so I move them out of this block and place them here.

**Comment by CaitinChen** [comment]
@lilin90 @amyangfei Thanks. PTAL again.

**Pull request merged**